routers/usersRouter.js

```diff
@@ -0,0 +1,90 @@
+const express = require('express');
```

Comment:
great job on the code here. 
I love the commenting to keep things organized and easy to find